piano-hero/core/src/main/java/com/novoda/pianohero/Notes.java

```diff
@@ -0,0 +1,58 @@
+package com.novoda.pianohero;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+public class Notes implements Iterable<Note> {
+
+    public static final Notes EMPTY = new Notes(Collections.<Note>emptySet());
+
+    private final Set<Note> notes;
```

Comment:
The utility of the set is to ensure no duplicates - but apart from that, it's a bit annoying because it has no `get(int)` methods...